### PR TITLE
get user geolocation and add marker to restaurant info page

### PR DESCRIPTION
**Pull request by Lynne-Daniels**

### CODE REVIEW

File: client/src/components/customer/GoogleMap.jsx [hunk 1]
```diff
@@ -58,6 +73,14 @@ export default class GoogleMap extends Component {
           text={this.state.restaurantName}
           src="http://maps.google.com/mapfiles/kml/paddle/red-stars.png"/>
 
+          <img position="absolute"
+            height="33"
+            width="33"
+            lat={this.state.currentLocation.lat}
+            lng={this.state.currentLocation.lng}
+            text="You are Here"
+            src="http://maps.google.com/mapfiles/kml/paddle/red-stars.png"/>
```
Comment (lamdbui) [inline client/src/components/customer/GoogleMap.jsx]: Could we use a different icon here? Maybe a person icon(or something similar) to differentiate between the restaurant location and the user location?

**Review by lamdbui — CHANGES_REQUESTED** [review]
Looks good. There's just the icon change I suggested below.

**Review by ryanakiyama — COMMENTED** [review]
Agreed to change icon

**Review by ryanakiyama — APPROVED** [review]
Looks good

**Pull request merged**